server/Dockerfile

```diff
@@ -17,13 +17,13 @@ RUN npm run build
 RUN npm prune --omit=dev --omit=optional
 
 
-FROM node:18-alpine3.17
+FROM node:18.16.0-alpine3.18@sha256:f41850f74ff16a33daff988e2ea06ef8f5daeb6fb84913c7df09552a98caba09
 
 ENV NODE_ENV=production
 
 WORKDIR /usr/src/app
 
-RUN apk add --no-cache vips-heif vips vips-cpp ffmpeg perl
+RUN apk add --no-cache ffmpeg imagemagick-dev libraw-dev perl vips-dev vips-heif vips-magick
```

Comment:
`-dev` packages may not be needed here.

Reply:
I tested without `vips-dev` and it wasn't happy at all. I presume all it wants is `vips-cpp`.

Reply:
Nope, not happy with `vips-cpp` either. I'll leave the `-dev` packages as is :)